python/dgl/graphbolt/impl/torch_based_feature_store.py

```diff
@@ -85,6 +85,17 @@ def read(self, ids: torch.Tensor = None):
             return self._tensor
         return self._tensor[ids]
 
+    def size(self):
+        """Get the size of the feature.
+
+        -------
+        torch.Size
+            The size of the feature.
+        """
+        if len(self._tensor.shape) == 1:
```

Comment:
We don't need this anymore, right?

Reply:
Right. I just merge the `master` branch into this branch. I am modifying the class and the tests.